data/categories.txt

```diff
@@ -7241,7 +7243,7 @@ pl:^Sklep monopolowy
 pt:^Vinho|Loja de Vinhos
 pt-BR:^Loja de vinhos|bebidas
 ro:^Magazin de băuturi alcolice
-es:^Establecimiento de bebidas alcohólicas
+es:^Vino|Tienda de vinos
```

Comment:
Note: the "^" term will be used in the UI. Depending on what's actually is written on most signboards, we probably should swap these two synonyms.